framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/AbpDbContext.cs

```diff
@@ -204,6 +204,7 @@ protected virtual void TrySetDatabaseProvider(ModelBuilder modelBuilder)
             case "Npgsql.EntityFrameworkCore.PostgreSQL":
                 return EfCoreDatabaseProvider.PostgreSql;
             case "Pomelo.EntityFrameworkCore.MySql":
+            case "MySql.Data.MySqlClient":
```

Comment:
<img width="1193" height="188" alt="image" src="https://github.com/user-attachments/assets/aa58621f-eead-45bc-bac0-e6fe6c660720" />
<img width="1179" height="49" alt="image" src="https://github.com/user-attachments/assets/27f54c67-c666-4568-adb7-e422e213bb1d" />

The name that is actually obtained is `MySql.EntityFrameworkCore`.